### ISSUE TITLE
Attribute "borderWidth" has already been defined

### ISSUE THREAD

**Issue by ryandt** [Innovattic/Android-Font-Library#8]
When adding "compile 'com.android.support:design:22.2.0'" to my build.gradle file, I am receiving the error "Attribute "borderWidth" has already been defined" at compile time. There appears to be a naming collision with Android-Font-Integration's borderWidth and the Floating Action Button's borderWidth.

**Comment by jfresen** [comment]
Hi, I've confirmed the issue and will fix it asap, probably over the weekend. The fix will consist of prefixing the attributes in this library with "fl" (for font-library) less likely to collide with other libraries, although it is by no means a guarantee.

If you have an idea for a better solution, please let me know. If you think a different prefix is more intuitive, also let me know.

**Comment by ryandt** [comment]
Prefixing your XML attributes is the most common solution. I recommend using an acronym of your library as the prefix (e.g. "fi_" or "fil_").

Cheers

**Comment by jfresen** [comment]
Then I should probably change the name of the project. I named it after the title of an accompanying blog post, to showcase what I explained therein, but it has since grown to an actual library. android-font-library better describes what it is, the prefix will then become "fl_" or "fl". Personally, I am more inclined to use camel case attribute names, as Android tends to use camel case as well. I noticed I have been inconsistent in that respect though, as the id font_extra_data is not in that format. I will change that name to flFontExtraData to maintain consistency. Changes will be applied during the weekend. Thanks for your input!
